data/autoyast_sle15/autoyast_sle-micro.xml

```diff
@@ -2,9 +2,9 @@
 <!DOCTYPE profile>
 <profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
   <suse_register>
-    <do_registration config:type="boolean">false</do_registration>
+    <do_registration config:type="boolean">true</do_registration>
     <reg_code>{{SCC_REGCODE}}</reg_code>
-    <install_updates config:type="boolean">false</install_updates>
+    <install_updates config:type="boolean">true</install_updates>
```

Comment:
is the UI installation doing this?

Reply:
ooops, over here it does not make sense as it is used only for slem under dev.